src/shared/tests/pipes.test.tsx

```diff
@@ -84,21 +83,9 @@ describe('Formatting lists tests', () => {
 
 describe('Formatting End Date Tests', () => {
   test('with dummy data', () => {
-    expect(endDatePipe(exampleWorkPackage1.startDate, exampleWorkPackage1.duration)).toBe(
```

Comment:
Here is the area where I removed some tests referenced in ticket...I believe most of them should work(that don't use endDatePipe)